tests/sentry/utils/performance_issues/test_performance_detection.py

```diff
@@ -174,6 +175,36 @@ def test_system_option_used_when_project_option_is_default(self):
             perf_problems = _detect_performance_problems(n_plus_one_event, sdk_span_mock)
             assert_n_plus_one_db_problem(perf_problems)
 
+    @override_options({"performance.issues.n_plus_one_db.problem-creation": 1.0})
+    def test_respects_organization_creation_permissions(self):
+        n_plus_one_event = EVENTS["n-plus-one-in-django-index-view"]
```

Comment:
We prefer to use `get_event` here now to avoid accidentally mutating data.

Reply:
Done in [d82bb36](https://github.com/getsentry/sentry/pull/42777/commits/d82bb3631592f079cbf1254db88f85a1007e864e)!